front-end/src/common/stats-generators/CSPMStatGenerator.ts

```diff
@@ -1,6 +1,6 @@
 import {BaseStatGenerator} from "./BaseStatsGenerator";
-import PlayerStatModel from "../../../../../../Common/models/playerstat.model";
-import {riotTimestampToMinutes} from "../../../../../../Common/utils";
+import PlayerStatModel from "../../../../Common/models/playerstat.model";
+import {riotTimestampToMinutes} from "../../../../Common/utils";
```

Comment:
You MUST order imports by the ones with {} first then worded ones second!!!!

Reply:
😔🔫